### DISCUSSION TITLE
Advice for plotting 3D hull in OrbitView

### DISCUSSION THREAD

**Discussion by cornhundred** [visgl/deck.gl#5413] [Q&A] [answered]
I would like to plot a 3D hull around a set of points with xyz coordinates. If I use a tool like https://github.com/swapnil96/Convex-hull in python to calculate the hull, how should I pass it to the polygon layer in deck.gl - individual polygons with xyz coordinates for each flat section of the 3D hull? 

Similarly, if I want to plot a 3D cube in OrbitView do I need to pass in six square polygons or can I combine some polygons together into a larger set of coordinates?

**Comment by Pessimistress** [comment, marked as answer]
[SimpleMeshLayer](https://deck.gl/docs/api-reference/mesh-layers/simple-mesh-layer)
